ddtrace/internal/ci_visibility/writer.py

```diff
@@ -151,6 +153,8 @@ def __init__(
             timeout=timeout,
             dogstatsd=dogstatsd,
             sync_mode=sync_mode,
+            # FIXME(munir): report_metrics is not used in the CI Visibility Writers
+            # self._report_metrics = report_metrics,
```

Comment:
`report_metrics` parameter is also never used but passing this parameter to the parent class breaks tests. I don't want to fix these tests in this PR so I just left a comment. This is worth looking into